libdirections-hybrid/build.gradle

```diff
@@ -21,7 +21,7 @@ android {
         minSdkVersion androidVersions.minSdkVersion
         targetSdkVersion androidVersions.targetSdkVersion
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        consumerProguardFiles 'proguard-consumer.pro'
+        consumerProguardFiles 'proguard-rules.pro', "${rootDir}/proguard/proguard-project.pro"
```

Comment:
Curious 🤔 Is Proguard merging rules automatically when you add different files separated by `,`? Didn't know 😅

Reply:
that's right 😉

Reply:
have a look at signature `public void consumerProguardFiles(@NonNull Object... proguardFiles)` 👀